nautobot/extras/tests/test_views.py

```diff
@@ -881,6 +881,8 @@ def test_only_enabled_is_listed(self):
         self.assertHttpStatus(response, 200)
         self.assertNotIn("test4", extract_page_body(response.content.decode(response.charset)))
 
+    # TODO: add `def test_non_valid_crontab_syntax(self):`
```

Comment:
This is looking great. I would like to see some tests here as we had indicated as what happens when `ScheduledJob.get_crontab()` goes wrong.

And then also asserting that we can correctly create Crontab-based scheduled jobs via the API as well as the UI.